3.10/administration-cluster.md

```diff
@@ -20,45 +20,59 @@ Enabling synchronous replication
 For an introduction about _Synchronous Replication_ in Cluster, please refer
 to the [_Cluster Architecture_](architecture-deployment-modes-cluster-architecture.html#synchronous-replication) section. 
 
-Synchronous replication can be enabled per _collection_. When creating a
-_collection_ you may specify the number of _replicas_ using the
-*replicationFactor* parameter. The default value is set to `1` which
-effectively *disables* synchronous replication among _DB-Servers_. 
+You can enable synchronous replication per _collection_. When you create a
+_collection_, you may specify the number of _replicas_ using the
```

Comment:
I would add an explicit comment here, that this number includes the leader and all followers. This is a common misunderstanding and cannot be emphasized often enough. Something along the lines of:
"The number of replicas includes the leader (the master copy) as well as all the followers (redundancy copies)."
Just a suggestion, maybe you find this clear enough, then just feel free to ignore.

Reply:
Good call!